code/datums/emotes.dm

```diff
@@ -0,0 +1,353 @@
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_VOICELESS
+
+/datum/emote/twitch
+	id = "twitch"
+	message_targeted = "#SOURCE flinches."
+	message_untargeted = "#SOURCE twitches."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/gasp
+	id = "gasp"
+	message_targeted = "#SOURCE gasps at #TARGET."
+	message_untargeted = "#SOURCE gasps."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/gasp_heavy
+	id = "gasp_heavy"
+	message_targeted = "#SOURCE gasps for air!"
+	message_untargeted = "#SOURCE gasps for air!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/yawn
+	id = "yawn"
+	message_targeted = "#SOURCE yawns at #TARGET."
+	message_untargeted = "#SOURCE yawns."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/snore
+	id = "snore"
+	message_targeted = "#SOURCE sarcasticly snores at #TARGET."
+	message_untargeted = "#SOURCE snores."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/doze
+	id = "doze"
+	message_targeted = "#SOURCE has trouble keeping their eyes open."
+	message_untargeted = "#SOURCE has trouble keeping their eyes open."
+	display_flags = EMOTE_VISIBLE | EMOTE_DISALLOW_MACHINE
+
+/datum/emote/limp
+	id = "limp"
+	message_targeted = "#SOURCE's body becomes limp."
+	message_untargeted = "#SOURCE's body becomes limp."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/bounce
+	id = "bounce"
+	message_targeted = "#SOURCE bounces towards #TARGET."
+	message_untargeted = "#SOURCE bounces in place."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/sway
+	id = "sway"
+	message_targeted = "#SOURCE sways towards #TARGET."
+	message_untargeted = "#SOURCE sways."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/light
+	id = "light"
+	message_targeted = "#SOURCE lights up at the sight of #TARGET."
+	message_untargeted = "#SOURCE lights up."
+	display_flags = EMOTE_VISIBLE //"Lights up" is pretty ambiguous. Could mean literally or metaphorically.
+
+/datum/emote/vibrate
+	id = "vibrate"
+	message_targeted = "#SOURCE vibrates towards #TARGET."
+	message_untargeted = "#SOURCE vibrates."
+	display_flags = EMOTE_VISIBLE | EMOTE_DISALLOW_ORGANIC
+
+/datum/emote/jiggle
+	id = "jiggle"
+	message_targeted = "#SOURCE jiggles their breasts at #TARGET." //DO YOU THINK THEY'LL NOTICE?
+	message_untargeted = "#SOURCE jiggles."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/alarm
+	id = "alarm"
+	message_targeted = "#SOURCE hails #TARGET with an alarm!"
+	message_untargeted = "#SOURCE sounds off an alarm!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_ORGANIC
+
+/datum/emote/beepboop
+	id = "beepboop"
+	message_targeted = "#SOURCE akes an excited beeping booping sound at #TARGET!"
+	message_untargeted = "#SOURCE akes an excited beeping booping sound!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_ORGANIC
+
+/datum/emote/notice
+	id = "notice"
+	message_targeted = "#SOURCE emits a noticeable beep towards #TARGET."
```

Comment:
All beeps are noticeable. If it wasn't noticeable then it wouldn't be noticed - redundant.